FirstProject/css/feedback.css

```diff
@@ -89,15 +78,16 @@ h1.interview_text_area {
     }
 
     .left_image {
-        float: left!important;
+        width: 100%;
     }
 
     header form {
         margin-left: 0!important;
     }
 
     .header_form_section {
-        width: 94%!important;
+        width: 87%;
+        margin: 0 40px auto auto;
```

Comment:
Just making sure that this is what you want it to be: can you describe what this expands to?